AWS/IAM/TPlaybook_delete_IAMUser.md

```diff
@@ -0,0 +1,31 @@
+
+<img src="../../images/icons/Tamnoon.png" width="200"/>
+
+# Tamnoon Playbook: IAM - Delete IAMUser .
+
+## Description
+This playbook describes how to execute Tamnoon IAMHelper automation to delete iam user.
+
+The execution is based on AWS credentials configuration based on the next fallbacks:
```

Comment:
should be "authentication" and not "execution". The whole sentence should be: The authentication process for this playbook follows follows the standard AWS set of fallbacks: